src/utils/generateQRCode.js

```diff
@@ -0,0 +1,14 @@
+import qrcode from 'qrcode';
+import crypto from 'crypto';
+
+export async function generatingQRCode() {
```

Comment:
Por convenção, nomes de funções devem ser verbos no presente simples. Esta aqui poderia ser renomeada.

Além disso, o QRCode deverá ser gerado com base no número de matrícula do estudante (`studentId`) e não em uma string aleatória.

```suggestion
export async function generateStudentQRCode(studentId) {
```

Reply:
Sim, eu disse isso para o GV, mas como ele irá criar um token propio e não será usado apenas o número de matrícula, então avisei a ele que iria deixar sim

Reply:
Acho que você esqueceu de renomear a função